readers/docker/containers_cpu_linux.go

```diff
@@ -55,7 +55,7 @@ func (m *DockerContainersCpu) Run() error {
 
 			data, err := gopsutil_docker.CgroupCPU(containerDir, m.CgroupBasePath)
 			if err == nil {
-				m.Data[container.Image+"-"+container.ID] = data
```

Comment:
What I'm using is the name of the container, not the image. The docker docs say that the container name is guaranteed to be unique. I'm pretty sure it's also required. This makes it an ideal hash key, no?